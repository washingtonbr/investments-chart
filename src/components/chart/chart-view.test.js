import React from 'react';
import renderer from 'react-test-renderer';
import Chart from './chart-view';
import {
  mockChartType,
  mockChartData,
  mockChartOptions,
} from './chart-view.mock';

describe('Chart', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Chart
          type={mockChartType}
          data={mockChartData}
          options={mockChartOptions}
        />,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
