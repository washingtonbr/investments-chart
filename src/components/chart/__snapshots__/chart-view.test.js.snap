// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart renders correctly 1`] = `
<canvas
  height={55}
  width={100}
/>
`;
